### PR TITLE
fix swap transactions and show matic/usdt pair chart by default on swap pro mode

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

File: src/apollo/client.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const client = new ApolloClient({
 
 export const txClient = new ApolloClient({
   link: new HttpLink({
-    uri: process.env.REACT_APP_TX_GRAPH_API_URL,
+    uri: process.env.REACT_APP_GRAPH_API_URL,
```
Comment (FriskyHamTitz) [inline src/apollo/client.js:15]: Why did this change?

File: src/apollo/queries.ts [hunk 2]
```diff
@@ -687,6 +687,7 @@ export const SWAP_TRANSACTIONS = gql`
       first: 1000
       where: { pair_in: $allPairs, timestamp_gte: $lastTime }
       orderBy: timestamp
+      orderDirection: desc
```
Comment (FriskyHamTitz) [inline src/apollo/queries.ts:690]: What does this have to do with pro mode

File: src/pages/SwapPage/SwapPage.tsx [hunk 3]
```diff
@@ -70,10 +70,11 @@ const SwapPage: React.FC = () => {
         if (txns) {
           const filteredTxns = txns.filter(
             (txn) =>
-              !transactions.filter(
+              !transactions.find(
                 (tx) => tx.transaction.id === txn.transaction.id,
-              ).length,
+              ),
           );
+          console.log('ccc', filteredTxns);
```
Comment (FriskyHamTitz) [inline src/pages/SwapPage/SwapPage.tsx:77]: Remove this log message and the other one that says bbb which got merged to master

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]
Few minor comments

File: src/pages/SwapPage/SwapPage.tsx (hunk 3 above)
Comment (totop716) [reply src/pages/SwapPage/SwapPage.tsx]: Removed the console logs and pushed commit

File: src/apollo/client.js (hunk 1 above)
Comment (totop716) [reply src/apollo/client.js:15]: The graph does not work and made this change to get transactions from furadao

File: src/apollo/queries.ts (hunk 2 above)
Comment (totop716) [reply src/apollo/queries.ts:690]: This is to put recent transactions at first.

**Review by FriskyHamTitz — APPROVED** [review]
